.gitignore

```diff
@@ -0,0 +1 @@
+.vscode/settings.json
```

Comment:
Unnecessary addition to the PR. Let's remove this.

Reply:
Removed it.